packages/tool-cache/src/tool-cache.ts

```diff
@@ -435,7 +438,7 @@ export async function cacheDir(
   // due to anti-virus software having an open handle on a file.
   for (const itemName of fs.readdirSync(sourceDir)) {
     const s = path.join(sourceDir, itemName)
-    await io.cp(s, destPath, {recursive: true})
+    await io.cp(s, destPath, {recursive: true, preserveTimestamps: preserveTimestamps})
```

Comment:
Probably want the same treatment for `cacheFile` for consistency

Reply:
Good call. Done - https://github.com/actions/toolkit/pull/1617/commits/5702324552e377d96f5e746f38b33c62e7f25045